mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ContainerEventHandlerAdapterBuilder.java

```diff
@@ -1,8 +1,9 @@
 package org.wildfly.extension.mod_cluster;
 
+import org.jboss.as.controller.OperationContext;
 import org.jboss.msc.service.ServiceBuilder;
 import org.jboss.msc.service.ServiceTarget;
 
 public interface ContainerEventHandlerAdapterBuilder {
-    ServiceBuilder<?> build(ServiceTarget target, String connector, int statusInterval);
+    ServiceBuilder<?> build(ServiceTarget target, OperationContext context, String connector, int statusInterval);
```

Comment:
there is already CapabilityServiceBuilder in core nowdays and most of this PR uses it.
UndertowEventHandlerAdapterBuilder which is the only impl of this interface is also changed as part of this PR, so that shouldn't be an issue